components/layout/query.rs

```diff
@@ -780,15 +843,20 @@ where
             layout_el: <N::ConcreteThreadSafeLayoutNode as ThreadSafeLayoutNode>::ConcreteThreadSafeLayoutElement,
             layout_root: &mut Flow,
             requested_node: N,
-            longhand_id: LonghandId) -> String {
```

Comment:
Rustfmt really struggled here. It wants to format the line as `longhand_id: LonghandId) -> String{` without any indentation. I manually formatted it so both rustfmt and tidy accept it.

Reply:
Could you recreate a minimal testcase and submit an issue to https://github.com/rust-lang-nursery/rustfmt/?